### ISSUE TITLE
Missing A11Y, Geo and Text specific glyphs on my end.

### ISSUE THREAD

**Issue by CrisA-Works** [calendso/font#28]
First of all, the latest update improved the performance and crashing issues I had! Yay! It's still a heavy typeface to use, since my Ai slows down when zooming in an out, but I think that's more expected from huge VF.

I can finally do some testing. Here's what I was trying to showcase but I wasn't able to.

It seems that on my end the A11Y, Geo and Text specific glyphs aren't appearing. Here's a PDF showcasing which ones I can currently use. You might want to download the file since opening it in the browser through the link gives a funky preview that isn't in the actual file.

[Pruebas Cal Sans.pdf](https://github.com/user-attachments/files/29224295/Pruebas.Cal.Sans.pdf)

On my end all A11Y, Geo and Text (except for Text `f` and `t`) look like the UI variants. For the individual letters I used the fixed options within the variable file, like Geo Regular for example.

I added the Text A11Y, Text Geo and Text UI next to my testing just in case. I'm not sure what to look for from those sub variants.

**Comment by MarkFonts** [comment]
this looks like an old build of the font conflicting with the new

**Comment by MarkFonts** [comment]
Might be simpler testing using the static families?
[A11y](https://github.com/calcom/sans/tree/main/fonts/calsans-static-a11y)
[UI](https://github.com/calcom/sans/tree/main/fonts/calsans-static-ui)
[Base](https://github.com/calcom/sans/tree/main/fonts/calsans-static-base)
[Geo](https://github.com/calcom/sans/tree/main/fonts/calsans-static-geo)

Sorry to hear software performance took a hit using the font. Does Adobe Illustrator performance get better with this [variable build? no stylistic sets](https://github.com/calcom/sans/blob/main/fonts/calsans-cossui/CalSansVF.ttf).

Can you confirm the performance is still worsened with only illustrator open and no, say, internet browser or photoshop applications holding some performance?

**Comment by MarkFonts** [comment]
@CrisA-Works have you tried removing all cal sans builds from your computer/font managers, [clearing your font cache](https://www.google.com/search?q=clearing+font+cache&oq=clearing+font+c&gs_lcrp=) then seeing if the font works ok?

**Comment by CrisA-Works** [comment]
I don't think that FontBase installs the typeface anywhere, it loads the local file and un-loads it after the app is either closed or you de-activate it yourself. Still I did a quick search and I couldn't find any folder that could be keeping a copy of the typefaces it loads.

Tried clearing font cache and using the VF without stylistc sets, but the issue persists.

I can't test the static versions because Fontbase interprets them all as duplicates from Cal Sans - Regular

<img width="1383" height="1039" alt="Image" src="https://github.com/user-attachments/assets/f018fc95-b2f4-43df-88d2-33b73bad7dec" />

**Comment by CrisA-Works** [comment]
Could this be the cause? The static file I'm currently using is the A11Y one, but it still appears as Cal Sans Regular

<img width="662" height="362" alt="Image" src="https://github.com/user-attachments/assets/9e1054dc-7e13-4bfc-829d-56ae91777d27" />

Commissioner separates its typefaces very cleanly because seemingly it has this meta-data well sorted

<img width="677" height="540" alt="Image" src="https://github.com/user-attachments/assets/18d78ba8-60f3-42c4-a177-ca5fe48c14d5" />

<img width="711" height="619" alt="Image" src="https://github.com/user-attachments/assets/8b2dfdb8-12e3-4794-99f5-75543a07b558" />
